src/php/install.sh

```diff
@@ -169,7 +169,7 @@ install_php() {
         VERSION_CONFIG="--with-pear"
     fi
 
```

Comment:
Can you help bump the minor version in here? thanks! 

https://github.com/devcontainers/features/blob/2b778155760bf3e248bce5ca998fcc487ec8cd80/src/php/devcontainer-feature.json#L3